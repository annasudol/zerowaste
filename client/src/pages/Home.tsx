import * as React from 'react';
import { Image, SearchRecipesForm } from '../components';
const fridge = require('./../assets/images/svg/technology.svg');

export const Home: React.FC = () => (
    <div className='content overflow-hidden'>
        <div className='flex flex-col justify-center items-center md:flex-row-reverse items-center h-screen'>
            <div className='mb-8 text-center md:w-1/3 lg:w-1/4 md:text-left min-h-32'>
                <h1 className='font-bebas uppercase text-darkGray mb-3'>your spare food</h1>
                <SearchRecipesForm />
            </div>
            <Image src={fridge.default} alt='fridge' size='medium' />
        </div>
    </div>)