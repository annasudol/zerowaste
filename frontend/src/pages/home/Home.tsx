import React, { FC, ReactElement } from 'react';
import { Image, SearchRecipesForm } from '../../components';
const fridge = require('./../../assets/images/svg/technology.svg');

export const Home: FC = (): ReactElement => (
            <>
                <div className='flex flex-col justify-center items-center md:flex-row-reverse items-center h-screen'>
                    <div className='mb-8 text-center md:w-1/3 lg:w-1/4 md:text-left'>
                        <h1 className='font-bebas uppercase text-darkGray mb-3'>your spare food</h1>
                        <SearchRecipesForm />
                    </div>
                    <Image src={fridge} alt='fridge' size='medium' />
                </div>
            </>
       )