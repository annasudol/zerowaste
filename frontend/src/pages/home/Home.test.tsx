import { shallow } from 'enzyme';
import React from 'react';
import DeleteIcon from '@material-ui/icons/Delete';
import { Home } from './Home';
import toJson from 'enzyme-to-json';
import { act } from "react-dom/test-utils";

describe('component', (): void => {
  describe('Home', (): void => {
        it('match snapshot', (): void => {
        const wrapper = shallow(
            <Home />,
        );

        expect(toJson(wrapper)).toMatchSnapshot();
        });
    });
});
