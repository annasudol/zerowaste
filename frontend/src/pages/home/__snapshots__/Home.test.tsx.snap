// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component Home match snapshot 1`] = `
<Fragment>
  <div
    className="flex flex-col justify-center items-center md:flex-row-reverse items-center h-screen"
  >
    <div
      className="mb-8 text-center md:w-1/3 lg:w-1/4 md:text-left"
    >
      <h1
        className="font-bebas uppercase text-darkGray mb-3"
      >
        your spare food
      </h1>
      <SearchRecipesForm />
    </div>
    <Image
      alt="fridge"
      size="medium"
      src={
        Object {
          "ReactComponent": Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          },
          "__esModule": true,
          "default": "technology.svg",
        }
      }
    />
  </div>
</Fragment>
`;
